R/plumber.R

```diff
@@ -902,6 +876,54 @@ plumber <- R6Class(
     setErrorHandler = function(fun){
       private$errorHandler <- fun
     },
+    #' @description Add a function to customize what is returned in `$apiSpec()`.
+    #' @param api_fun This function should accept an OpenAPI Specification autogenerated by `plumber`.  It should return a list object in valid OpenAPI Specification format. This will not be validated.
+    apiSpecTransform = function(api_fun = NULL) {
+      if (is.null(api_fun)) {
+        api_fun <- identity
+      }
+      if (!is.function(api_fun)) {
+        stop("`api_fun` must be a function")
+      }
+      private$apiHandler <- api_fun
+    },
+    #' @description Set UI to use for API
+    #' @param ui a character value or a logical value. Default to `plumber.ui` option value.
```

Comment:
There's not a list of valid values in here, nor a hint as to where to find them.